.github/workflows/esti.yaml

```diff
@@ -1184,13 +1184,22 @@ jobs:
       AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
       AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
       AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      COSMOSDB_DATABASE: esti-db
+      COSMOSDB_ACCOUNT: esti-e2e-tests
+      COSMOSDB_CONTAINER_PREFIX: ${{ github.run_number }}
+
     steps:
+      - name: Azure log-in
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+        
       - name: Check-out code
         uses: actions/checkout@v3
 
       - name: Generate uniquifying value
         id: unique
-        run: echo "value=$RANDOM" >> $GITHUB_OUTPUT
+        run: echo "cosmos_db_container=${{ env.COSMOSDB_CONTAINER_PREFIX }}-$RANDOM" >> $GITHUB_OUTPUT
```

Comment:
- I would keep the same value=RANDOM pattern and not break it.
- Now that we generate unique names each run the chance that we leave something is bigger. We can have a step before that run the same clean command also before in case this is a rerun if we worry about previous clean that failed.
- The value `${{ github.run_number }}-${{ steps.unique.outputs.value }}` that is used for a container name can be used for both `LAKEFS_DATABASE_COSMOSDB_CONTAINER` and the drop comment.